src/assembly/agent-load-test/agent-load-test.bat

```diff
@@ -0,0 +1,113 @@
+@REM ----------------------------------------------------------------------------
+@REM  Copyright 2001-2006 The Apache Software Foundation.
```

Comment:
This is just for a tool that we can distribute to users who want to test their own system limits